### DISCUSSION TITLE
We are planning to use assignmetadata mutation to add annotation to few namespaces which are in *-dev pattern.

### DISCUSSION THREAD

**Discussion by chaitushiva** [open-policy-agent/community#51] [Gatekeeper]
We do not want to use namespace as described in the doc(https://github.com/open-policy-agent/gatekeeper/blob/97e3ae535ef59782de906f41ea24ec2618183611/website/docs/mutation.md) Is there any other way to achieve this?
namespaceSelector - filters resources by namespace selector

**Comment by peteroneilljr** [comment]
@chaitushiva Is it possible to describe a bit more about the limitations in your system that you're trying to work around?
